components/infobox/wikis/ageofempires/infobox_map_custom.lua

```diff
@@ -30,49 +30,42 @@ local TYPES = {
 	water = 'Water',
 }
 
-local _args
-
 function CustomMap.run(frame)
-	local customMap = Map(frame)
-	customMap.createWidgetInjector = CustomMap.createWidgetInjector
-	customMap.addToLpdb = CustomMap.addToLpdb
-	customMap.getWikiCategories = CustomMap.getWikiCategories
-	_args = customMap.args
-	_args.releasedate = _args.date
```

Comment:
Is it intended to remove this line?

Reply:
During testing i saw no changes in LPDB saving. Additionally i never saw a map page that actually set ``args.date``.